order/batch_verifier.go

```diff
@@ -203,7 +203,9 @@ func (v *batchVerifier) Verify(batch *Batch, bestHeight uint32) error {
 				),
 			}
 
-		case unitsFilled < ourOrder.Details().MinUnitsMatch:
+		case ourOrder.Details().AuctionType != BTCOutboundLiquidity &&
+			unitsFilled < ourOrder.Details().MinUnitsMatch:
```

Comment:
I think these lines are correct. I suppose that if the bid is for oubound liquidity only a single unit will be filled. I wound consider adding a comment.

Further, I wander whether a third case is necessary:
```
		case ourOrder.Details().AuctionType == BTCOutboundLiquidity &&
			unitsFilled == 1
```